SPEC.md

```diff
@@ -2618,6 +2619,13 @@ Given an array of optional values, `select_all` will select only those elements
 
 This function will return `false` if the argument is an unset optional value. It will return `true` in all other cases.
 
+## String basename(File)
```

Comment:
does it only work on a `File` or will it work on a `String` too ? (Or anything that coerces to a `File`)